templates/jobs/patch-cluster-job.yaml

```diff
@@ -48,7 +48,7 @@ spec:
             sizeLimit: 10Mi
         - name: cluster-config
           configMap:
-            name: graphdb-cluster-config-configmap
+            name: {{ .Values.graphdb.clusterConfig.existingClusterConfig | default "graphdb-cluster-config-configmap" }}
```

Comment:
I noticed that the patch job waits on the external proxy to be accessible, what if such service is not used?

Reply:
Although the chart allows you to set replicas: 0 for the cluster proxy, it doesn't support working without it, for now.

Reply:
So the clients would not be able to use it as they are not using proxies for the moment